wire-ios/Wire-iOS/Sources/Components/Settings.swift

```diff
@@ -180,6 +180,14 @@ class Settings {
             ExtensionSettings.shared.disableLinkPreviews = newValue
         }
     }
+    static var isClipboardEnabled: Bool {
+        get {
+            return SecurityFlags.clipboard.isEnabled ? true : ExtensionSettings.shared.isClipboardEnabled
+        }
```

Comment:
I don't really get that, can you explain?

Reply:
We don't need this for this fix. Removed it